docs-devsite/auth.actioncodesettings.md

```diff
@@ -46,6 +47,11 @@ android?: {
 
 ## ActionCodeSettings.dynamicLinkDomain
 
+> Warning: This API is now obsolete.
+> 
+> Firebase Dynamic Links is deprecated. Migrate to use Firebase Hosting link and use `linkDomain` to set a custom domain instead.
```

Comment:
Seems like something is missing from this part of the sentence:

"... Firebase Hosting link..."

Reply:
It's actually what we want to convey. We're telling developers to migrate from Firebase Dynamic Links to Firebase Hosting link; and use the new `linkDomain` field instead of the old `dynamicLinkDomain` field to set a custom domain. Do you have any suggestions to improve the sentence?

Reply:
"Firebase Dynamic Links" is the name of something specific.  But "Firebase Hosting link" isn't the name of anything.

Maybe this should be something like:
- "Instead, migrate to use Firebase Hosting deep linking..." or
- "Instead, migrate to use custom domain linking from Firebase Hosting..." or
- "Instead, deep link to your custom domain using Firebase Hosting..."

Maybe we can look at how this is being written about in the upcoming narrative guides on firebase.google.com?

Reply:
I changed it to "Firebase Dynamic Links is deprecated. Instead, use `linkDomain` to set a custom domain for mobile links.". Please let me know what you think.

Reply:
Also, added the link to FDL FAQ as you suggested. Thanks!